### PR TITLE
Properly re-throw errors in aspect loading.

### PR DESCRIPTION
**Pull request by katre**
Apparently a description is required.

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/skyframe/BuildTopLevelAspectsDetailsFunction.java [hunk 1]
```diff
@@ -88,13 +89,19 @@ public SkyValue compute(SkyKey skyKey, Environment env)
...
-        return null; // some aspects are not loaded
+      try {
+        // Then return a list of indenpendent aspects to be applied on the top-level targets
+        ImmutableList<AspectDetails> aspectsDetailsList =
+            getIndependentTopLevelAspects(env, topLevelAspectsClasses);
+        if (aspectsDetailsList == null) {
+          return null; // some aspects are not loaded
+        }
+        return new BuildTopLevelAspectsDetailsValue(aspectsDetailsList);
+      } catch (AspectCreationException e) {
```
Comment (mai93) [inline src/main/java/com/google/devtools/build/lib/skyframe/BuildTopLevelAspectsDetailsFunction.java:100]: Can we add this exception catch to `getIndependentTopLevelAspects` to be similar to `getTopLevelAspects`?

**Comment by mai93** [comment]
I am curious to know what cases the absence of this re-throw can affect? I can add a test for that to take care of it in the future as well.

File: src/main/java/com/google/devtools/build/lib/skyframe/BuildTopLevelAspectsDetailsFunction.java (hunk 1 above)
Comment (katre) [reply src/main/java/com/google/devtools/build/lib/skyframe/BuildTopLevelAspectsDetailsFunction.java]: Done, now they are more similar.

**Comment by katre** [comment]
I added a test that shows the case I observed: I was using an aspect from the command line and hadn't removed the `attrs`, so it couldn't be created, but the error message was swallowed.

**Review by mai93 — APPROVED** [review]

**Pull request closed without merging**